samples/drivers/video/capture/src/main.c

```diff
@@ -41,7 +36,7 @@ static inline int display_setup(const struct device *const display_dev, const ui
 	/* Set display pixel format to match the one in use by the camera */
 	switch (pixfmt) {
 	case VIDEO_PIX_FMT_RGB565:
-		ret = display_set_pixel_format(display_dev, PIXEL_FORMAT_BGR_565);
+		ret = display_set_pixel_format(display_dev, PIXEL_FORMAT_RGB_565);
```

Comment:
FYI: I wanted to make a glue driver that converts a `display.h` device into a `video.h` sink. :)